ros/lib/config.py

```diff
@@ -157,3 +157,4 @@ def build_endpoint_url(ep):
 CW_LOGGING_FORMAT = '%(asctime)s - %(levelname)s  - %(funcName)s - %(message)s'
 ROS_PROCESSOR_PORT = int(os.getenv("ROS_PROCESSOR_PORT", "8000"))
 ROS_API_PORT = int(os.getenv("ROS_API_PORT", "8000"))
+API_VERSION = 'v1'
```

Comment:
In future we may get more than one api version and both could be supported? can we depend on api paths maybe to get verion and actually show all deployed api versions?